tests/integration_tests/Test/PM/Integration/_data/cpf-merge/module.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Export generator="Cache" version="25">
+  <Document name="cpf-merge.ZPM">
+    <Module>
+      <Name>cpf-merge</Name>
+      <Version>0.0.1</Version>
+      <Packaging>module</Packaging>
+      <SourcesRoot>src</SourcesRoot>
+
+      <!-- By default, the CPF merge happens right before Initialize -->
+      <CPF Name="config-globals.cpf"/>
+
+      <!-- Optionally, make it happen after MakeDeployed -->
+      <CPF Name="actions.cpf" Directory="custom-cpf-directory" Phase="MakeDeployed" When="After"/>
```

Comment:
@isc-shuliu  Does mapping of this structure to the structure below happen automatically using XSLTs?
```
<CPF Name="actions.cpf">
<Attribute Name="Directory">custom-cpf-directory</Attribute>
<Attribute Name="Phase">MakeDeployed</Attribute>
<Attribute Name="When">After</Attribute>
</CPF>
```

I ask because its a bit confusing distinguishing between what is a valid property on ResourceReference vs in the ResourceProcessor class

cc @isc-tleavitt